ENIGMAsystem/SHELL/Graphics_Systems/General/GSprimitives.h

```diff
@@ -31,6 +31,7 @@ namespace enigma_user
     pr_trianglefan    = 6      // GL_TRIANGLE_FAN      D3DPT_TRIANGLEFAN
   };
 
+  unsigned draw_primitive_count(int kind, unsigned vertex_count);
```

Comment:
maybe use enum : int as type here?

Reply:
I would have to change it everywhere else we declare groups of constants. I can do that at a later point in time, it requires enum classes. We had not used those to this point because I believe they are a C++11 feature we couldn't do back in the day. For now this is fine, you can still use the constants, but later updating it to enum classes will add additional safety.